python-clusters/create-aks-cluster/cluster.py

```diff
@@ -440,6 +442,7 @@ def do_delete():
             future = clusters_client.managed_clusters.begin_delete(resource_group, cluster_name)
             return future.result()
         delete_result = run_and_process_cloud_error(do_delete)
+        logging.info("Cluster deletion results")
```

Comment:
there's no results a this point
```suggestion
        logging.info("Cluster deletion requested")
```